src/applications/personalization/profile360/util/connected-apps.js

```diff
@@ -3,7 +3,7 @@ export const mockConnectedApps = [
     id: '0oa3s6dlvxgsZr62p2p7',
     type: 'okta_redis_apps',
     attributes: {
-      title: 'Test App',
+      title: 'Apple Health',
```

Comment:
I changed this to a name that's included in the available connected apps to test the filtering function (this one should not appear under 'apps to connect to'